### PR TITLE
development/github-desktop: Updated for 3.3.6 Build 3

### PR DESCRIPTION
**Pull request by mac-a-r0ni**

### CODE REVIEW

File: development/github-desktop/github-desktop.SlackBuild [hunk 1]
```diff
@@ -27,7 +27,7 @@ cd $(dirname $0) ; CWD=$(pwd)
 PRGNAM=github-desktop
 SRCNAM=GitHubDesktop
 VERSION=${VERSION:-3.3.6}
-BUILD=${BUILD:-2}
+BUILD=${BUILD:-3}
 TAG=${TAG:-_SBo}
 PKGTYPE=${PKGTYPE:-tgz}
```
Comment (aclemons) [inline development/github-desktop/github-desktop.SlackBuild:33]: `ar x $CWD/${SRCNAM}-linux-$DEBARCH-${VERSION}-linux2.deb data.tar.xz`
further needs to change to 
`ar x $CWD/${SRCNAM}-linux-$DEBARCH-${VERSION}-linux3.deb data.tar.xz`?

**Comment by aclemons** [comment]
@sbo-bot: build x86_64 development/github-desktop

File: development/github-desktop/github-desktop.SlackBuild (hunk 1 above)
Comment (mac-a-r0ni) [reply development/github-desktop/github-desktop.SlackBuild:33]: Gah, yes and I should have updated  the README as well. I'll push the fix in a moment.

Comment (mac-a-r0ni) [reply development/github-desktop/github-desktop.SlackBuild:33]: pushed fix!

**Comment by willysr** [comment]
@sbo-bot: build x86_64 development/github-desktop

**Comment by willysr** [comment]
merged

**Pull request closed without merging**